docs/reference/4-metaed/ide-user-guide/common-problems-and-troubleshooting.md

```diff
@@ -1,4 +1,4 @@
-# MetaEd IDE - Common Problems and Troubleshooting
+# Common Problems and Troubleshooting
```

Comment:
The side bar was too busy with all of these "MetaEd IDE" headers coming right below "MetaEd IDE User Guide". People will know that they're in the "MetaEd" context when viewing these pages.